workos/users.py

```diff
@@ -503,14 +503,14 @@ def verify_email(
         }
 
         response = self.request_helper.request(
-            USER_VERIFY_EMAIL_PATH,
+            USER_VERIFY_EMAIL_CODE_PATH,
             method=REQUEST_METHOD_POST,
             headers=headers,
             params=payload,
             token=workos.api_key,
         )
 
-        return WorkOSUser.construct_from_response(response).to_dict()
+        return WorkOSUser.construct_from_response(response["user"]).to_dict()
```

Comment:
just to make sure we want to hard fail if for some reason the "user" key is missing right? (else we'd do .get('user', {})

Reply:
Yeah... Since this is our API response, I think it's better than having a silent issue that may be hard to debug/understand.